shared/src/types.rs

```diff
@@ -437,7 +450,9 @@ pub struct ListenPortOpts {
 
 #[derive(Debug, Clone, PartialEq, Eq, Args)]
 pub struct OverrideEndpointOpts {
-    /// The listen port you'd like to set for the interface
+    /// The external endpoint that you'd like the innernet server to broadcast
+    /// to other peers. The IP address may be unspecified, in which case the
```

Comment:
Should we mention that the port will still be used, even if you decide to use an unspecified IP address?